benchmarks/dynamo/ci_expected_accuracy/update_expected.py

```diff
@@ -73,10 +73,10 @@ def get_artifacts_urls(results, suites):
 
 
 def normalize_suite_filename(suite_name):
-    assert suite_name.find("inductor_") == 0
     subsuite = suite_name.split("_")[1]
     if "timm" in subsuite:
-        subsuite = f"{subsuite}_models"
+        subsuite = subsuite.replace("timm", "timm_models")
```

Comment:
this is annoying, why can't we have just called the original timm_models

Reply:
`suite_name` comes from the CI config name, which does not contain the suffix, `_models`. I would vote for doing it the other way, changing `timm_models.py` to  `timm.py`.